docs/en/ingest-management/elastic-agent/configuration/env/shared-env.asciidoc

```diff
@@ -560,4 +560,20 @@ contains your CA's certificate.
 
 // end::kibana-ca[]
 
+// tag::netinfo[]
+|
+[id="env-{type}-netinfo"]
+`NETINFO`
+
+| (bool) When `false`, disables `netinfo.enabled` parameter of `add_host_metadata` processor.
+Setting this to `false` is recommended for large scale setups where the host.ip and host.mac fields index size increases.
```

Comment:
Lets add some more details here around what the problem is -> tons of ip addresses, tons of storage needed, slower queries

Reply:
Done. Let me know if is ok

Reply:
I think we can still add more details here but lets do this as a follow up.